.env

```diff
@@ -0,0 +1 @@
+VITE_API_URL="http://localhost:8000/api"
```

Comment:
why using `VITE` naming convection?

Reply:
I dont think it will work according to the documentation for dev. I sholud have placed this in gitignore

Reply:
Can you check? it should

Reply:
We ave the option of customizing it. But we hhave to use a pefix

Reply:
Instead of VITE_ We ccan use VUE_APP